index.html

```diff
@@ -0,0 +1,317 @@
+              (The link to the back-end API can be found in the GitHub repo)
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>React</li>
+              <li>redux</li>
+              <li>Jest</li>
+              <li>API</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/gtime-frontend/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card weather">
+          <div class="card-description">
+            <h2>Weather App</h2>
+            <p class="white-description">
+              This is a responsive weather app made with JavaScript that displays the temperature, humidity and timezone for cities around the world.
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>SASS/CSS</li>
+              <li>JavaScript</li>
+              <li>WebPack</li>
+              <li>Babel</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Weather-App/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card conference">
+          <div class="card-description">
+            <h2>Conference page</h2>
+            <p class="white-description">
+              This is a sample conference page made using HTML and CSS.
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>SASS</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Conference-page/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card katchy">
+          <div class="card-description">
+            <h2>Katchy</h2>
+            <p class="white-description">
+              This is a website for recipes made with Ruby on Rails
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>Ruby</li>
+              <li>Ruby on Rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>ERB</li>
+              <li>RSPec</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/katchy/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
```

Comment:
The about me section should follow the provided design. You can use flexbox to properly position the elements of your webpage to resemble the provided design. Kindly check below ⬇️ 

**Yours**
![about section yours](https://user-images.githubusercontent.com/32361963/119724876-f40fe480-be77-11eb-9816-3a6bbc350336.PNG)

**it should be**

![about section original](https://user-images.githubusercontent.com/32361963/119724918-fffba680-be77-11eb-9a1f-cc3345efc912.PNG)